source/_integrations/openplantbook.markdown

```diff
@@ -0,0 +1,106 @@
+---
+title: OpenPlantbook
+description: Instructions on how to setup OpenPlantbook with Home Assistant.
+ha_category:
+  - Environment
+ha_iot_class: Cloud Polling
+ha_release: 0.116
+ha_quality_scale: no score
+ha_codeowners:
+  - '@Olen'
+ha_domain: openplantbook
+---
+
+The `openplantbook` component allows you to search and get data from the [OpenPlantbook API](https://open.plantbook.io/) 
+
+## Configuration
+
+The integration is set up using the GUI.  You must have a valid `client_id` and `secret` from OpenPlantbook to set up the integration.
```

Comment:
Here you could put some additional information about how to set up the integration in the GUI, there are now plenty of examples to be found with other integrations 😉

Reply:
Thanks for helping out.